frontends/common/options.cpp

```diff
@@ -154,6 +154,13 @@ CompilerOptions::CompilerOptions() : ParserOptions() {
             return true;
         },
         "Unrolling all parser's loops");
+    registerOption(
```

Comment:
I am not sure if adding an extra option is a good either. Either this should run as part of the default suite, or right after parserUnroll, if it depends on it.